pkgs/servers/gemini/gmid/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "gmid";
-  version = "1.8.1";
+  version = "1.8.3";
 
   src = fetchFromGitHub {
     owner = "omar-polo";
     repo = pname;
     rev = version;
-    hash = "sha256-XNif164C5b5sVsZW7sy0id4qM/mJzg3RhoHbwJuJqDk=";
+    hash = "sha256-vghoPsyGspPn22Kl61qiaALS2R243JSuS80uKFBHc9k=";
   };
 
   nativeBuildInputs = [ bison which ];
```

Comment:
Not sure, can you check whether `which` is still needed (https://github.com/omar-polo/gmid/commit/12fcba2f80034356e75d73380d22012fd72476a8, https://github.com/omar-polo/gmid/commit/9db5e7051ebb37a9078e79ca46daf7f8f8f3f963)?